flink-connectors/flink-connector-pulsar/src/test/java/org/apache/flink/connector/pulsar/testutils/runtime/PulsarRuntimeOperator.java

```diff
@@ -180,6 +180,10 @@ public void createTopic(String topic, int numberOfPartitions) {
         }
     }
 
+    public void createSchema(String topic, Schema<?> schema) {
+        sneakyAdmin(() -> admin().schemas().createSchema(topic, schema.getSchemaInfo()));
```

Comment:
These checked exceptions should be in some situation. But I `sneaky` them just because of the poor design in Pulsar API. We can leave it here and work through all the code to see which `sneaky` should be better removed.

Reply:
Thanks for the brief clarification, @syhily . I created FLINK-30109 to cover the issue. We either should add more documentation clarifying why we do this or removing it as it's a workaround that might raise questions to readers. Let's move the discussion into FLINK-30109. I'm curious about your view on that and why the Pulsar API is forcing us to do this exception transformation.